docs/sources/developers/plugins/plugin.schema.json

```diff
@@ -23,8 +28,8 @@
     },
     "category": {
       "type": "string",
-      "description": "Plugin category used on the Add data source page. Possible values are: `tsdb`, `logging`, `cloud`, `tracing`, and `sql`.",
-      "enum": ["tsdb", "logging", "cloud", "tracing", "sql"]
+      "description": "Plugin category used on the Add data source page. Possible values are: `tsdb`, `logging`, `cloud`, `tracing`, `sql` and `others`.",
+      "enum": ["tsdb", "logging", "cloud", "tracing", "sql", "others"]
```

Comment:
Plugin will default to Others if `category` is not set. If we add `others`, should we make the category required? Since it then would be an exhaustive list?

Reply:
I think this can be answered only if we know the purpose of the "category" field.  Otherwise, tags/keywords are more powerful than the category as the category will be more exhaustive as you said. So it should be either category/keywords should be mandatory.

Reply:
`category` determines where the data source is displayed within the list of data sources.
<img width="302" alt="Screen Shot 2020-08-18 at 07 43 06" src="https://user-images.githubusercontent.com/8396880/90474835-7bc90580-e126-11ea-9463-006421ac1b49.png">

Reply:
Looking at the code, it looks like `enterprise` and `other` (not `others`) should be added to the schema.

https://github.com/grafana/grafana/blob/5a2efe1f62959f95a549d427bd692bc36dbd3ec3/public/app/features/datasources/state/buildCategories.ts#L5-L13